tests/psa-client-server/psasim/test/kill_server.sh

```diff
@@ -7,6 +7,9 @@ set -e
 
 pkill psa_server || true
 
+# Remove temporary files
```

Comment:
I think @ronald-cron-arm was quite clear he wanted this called `kill_servers.sh` - can you check that this rename is okay?

Reply:
I was a bit undecided about this change: I prefer to have cleanup code in this script in instead of `all.sh`, but at the same time the script is also killing _all_ the servers at once. OTOH a second call to `pkill` wouldn't hurt, so I can also keep the name as plural if you prefer.